### PR TITLE
added code block titles where relevant & some IER instructions formatting

### PR DESCRIPTION
**Pull request by gtiwari-ibm**

### CODE REVIEW

File: docs/3-Create/Deploy/ier.mdx [hunk 1]
```diff
@@ -218,24 +217,24 @@ On the left side is a folder in FileNet Content Manager that is in a record-enab
 On the right side is a special object store that contains the record objects. This is called a File Plan Object Store.
 
 
-      Please make sure below object stores are created and mentioned configurations are in place.These are the prerequisite for IER Functaionality to work correctly.
+[Create the following objects stores](https://ibm-client-engineering.github.io/solution-filenet-aws/Create/Deploy/solution-deploy-operator#adding-object-stores) and configure them as mentioned. These are the prerequisite for IER Functaionality to work correctly.
```
Comment (Manas-Jena-IBM) [inline docs/3-Create/Deploy/ier.mdx:220]: This works

**Review by Manas-Jena-IBM — COMMENTED** [review]
I found mostly formatting to the content . Like below . 
Line # 63
• Auto-destroy
- Auto-destroy
Line # 67 
1. Navigate to IER desktop and click on Configuration Tab. Select _Advanced Dispostion Schedules_ from dropdown as shown below.
1. Navigate to IER desktop and click on Configuration Tab. Select _Advanced Dispostion Schedules_ from the dropdown:

**Comment by gtiwari-ibm** [comment]
made requested changes

**Review by Manas-Jena-IBM — CHANGES_REQUESTED** [review]
line# 154 - Please reword "Enabling Workflow Tab on Content Navigator" to Enabling web PD on ICN. We can add required screenshots to this section to follow easily 

Line # 156 Prerequsites should hold this 
1.Download Plugin from below url . Example url - host/port/peengine/plugins/CPEAppletsPlugin.jar 

2. Install PD Plugin - add or point to the instusctions we have on "how to add a plugin 
in ICN "

3. Add a section to **create custom menu** to show the web PD option in there . After this step Web PD should be visible on desktop menu where the "custom menu was added" 

OR 

we can alternatively say Web PD is no more supported where all the above steps would be obsolete.

**Review by kramerro-ibm — APPROVED** [review]

File: docs/3-Create/Deploy/workflowsetup.mdx [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
 
-#### Setup Steps
+### Setup Steps
+
+#### Creating a Workflow System and Connection Point within Filenet
 
-##### Creating a Workflow System and Connection Point within Filenet
+<Tabs groupId="GUIvCR">
+  <TabItem value="GUI" label="Graphical User Interface (acce)">
 1. Open up and log into the acce console
```
Comment (Manas-Jena-IBM) [inline docs/3-Create/Deploy/workflowsetup.mdx:29]: Reword like - Log in to ACCE

**Review by Manas-Jena-IBM — APPROVED** [review]
Looks good to me

**Pull request merged**